assets/js/components/mobile/MobileLayout.jsx

```diff
@@ -47,9 +109,33 @@ class MobileLayout extends Component {
             </div>
           )
         }
+
+        {this.state.showSurveyNotification && organization && !organization.survey_token_sent_at && (
```

Comment:
are the extra 2 conditions needed if we never set `showSurveyNotification` to true unless these two conditions already true?